lib/processCss.js

```diff
@@ -168,18 +179,16 @@ module.exports = function processCss(inputSource, inputMap, options, callback) {
 					}
 				}
 				return url;
-			}
-		}),
-		extractImports(),
-		modulesScope({
+			},
 			generateScopedName: function generateScopedName (exportName) {
 				return customGetLocalIdent(options.loaderContext, localIdentName, exportName, {
 					regExp: localIdentRegExp,
 					hashPrefix: query.hashPrefix || "",
 					context: context
 				});
-			}
+			},
 		}),
+		icssComposes,
 		parserPlugin(parserOptions)
```

Comment:
Since the contract logic changes with renaming the various `postcss-modules-*` to `postcss-icss-*` my gut feeling tells me, that this will require a few changes to that internal plugin (`parserPlugin`) aswell